src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -16,6 +16,13 @@ kernel = "@KERNELPATH@"
 image = "@IMAGEPATH@"
 machine_type = "@MACHINETYPE@"
 
+# Enable confidential guest support.
```

Comment:
Per the discussion we had in the use-case meeting today, it would be nice to add a comment here indicating that we are just reserving the feature in the configuration file, but that the whole feature is not enabled yet (in order to avoid confusing someone checking out the code and expecting it to already work)

Reply:
Sorry I don't understand you.

> we are just reserving the feature in the configuration file

this is not just a reservation, if you enable it and have the right setup (kernel and qemu) in your system, you will get a TD container 

> but that the whole feature is not enabled yet (in order to avoid confusing someone checking out the code and expecting it to already work)

again this will work with the right setup

Reply:
@devimc For clarification, the mention of reserved interfaces was about the `Container image service` configuration.

Reply:
Thanks @c3d 
I agree, we should make it clear that this is a placeholder while we're enabling the technology.
Same applies for the container image service section. @devimc Would you mind adding this to the PR please?

Reply:
done, thanks